src/Symfony/Component/Serializer/Tests/Normalizer/Features/CallbacksTestTrait.php

```diff
@@ -175,13 +182,6 @@ public function provideNormalizeCallbacks()
                         return $result;
                     },
                 ],
-                [new CallbacksObject('baz'), new CallbacksObject('quux')],
-                ['bar' => 'bazquux', 'foo' => null],
-            ],
-            'Count a property' => [
```

Comment:
looks like something went wrong when changing this trait (see the failing tests)